dev/mp-dev.sh

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env bash
```

Comment:
I think we could add a short explanation at the top of the file with an overview of the script. What it can be used for and an summary of the features.